modules/rest-api/pages/rest-cluster-rebalance.adoc

```diff
@@ -133,7 +133,7 @@ This confirms that `10.143.190.103` has been removed.
 Adding a node to a cluster is a two-step process.
 
 . The `POST /controller/addNode` HTTP method and URI are used to add the node.
-This allows service-deployment for the node to be specified.
+This allows data service-deployment for the node to be specified.
```

Comment:
The previous line was correct.
When you use addNode, you are adding a node, so you need to be able specify all service-deployment for the node to be specified (not just data).  So, need to remove "data":

This allows service-deployment for the node to be specified.